modules/core/src/webgl-context/index.js

```diff
@@ -1,6 +1,6 @@
 export {createGLContext, destroyGLContext, resizeGLContext, pollGLContext} from './context';
 
-export {withParameters, resetParameters} from './context-state';
+export {withParameters, resetParameters, setParameters} from './context-state';
```

Comment:
Interesting, I wonder why this was changed? 
- Does LGTM just try to make sure all exports are forwarded? 
- Or is it a result of a multi-file/multi-depth analysis that concluded this was imported somewhere else?

Reply:
The [LGTM alert I found](https://lgtm.com/projects/g/uber/luma.gl/alerts/?mode=tree&ruleFocus=10030087) for this issue showed that setParameters was not a function and had a type of undefined. From that I was able to trace the error to the index file which was not exporting setParameters.

Reply:
That sounds great, definitely very useful